src/Symfony/Component/HtmlSanitizer/Visitor/Node/NodeInterface.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HtmlSanitizer\Visitor\Node;
+
+/**
+ * A node represents the sanitized version of a DOM node in the sanitized tree.
```

Comment:
Represents the sanitized version of a DOM node in the sanitized tree.

the rest after a blank line

all phpdocs should have a title + an optional description. right now, maybe use a long title instead